editor/js/components/arcontroller.component.js

```diff
@@ -153,7 +153,9 @@ ArControllerComponent.prototype.startAR = function() {
                 // Whenever ARController#process detects a marker, it fires a getMarker event
                 // with the marker details.
                 this.arController.addEventListener('getMarker',this.onTrackableFound.bind(this));
-                this.arController.addEventListener('getNFTMarker',this.onTrackableFound.bind(this));
+                document.addEventListener('sendMatrix', function (ev) {
```

Comment:
i attached the addEventListener to the document, it works but wondering if it is correct.